orangecontrib/spectroscopy/widgets/owpreprocess.py

```diff
@@ -1707,7 +1707,8 @@ def _initialize(self):
             # will be triggered by LayoutRequest event on the `flow_view`)
             self.__update_size_constraint()
 
-        self.apply()
+        # call apply() to output the preprocessor even if there is no input data
+        self.unconditional_commit()
```

Comment:
See my reply in #285, but even calling unconditional_commit() won't be sufficient to output the preprocessor even if there is no input data.

Reply:
That change was not meant to solve your problem. It was just a description for future us of what is being done, but it did not change anything really, except calling `apply()` once instead of twice on load.